superbuild/CMakeLists.txt

```diff
@@ -124,6 +131,34 @@ if (EXTERNAL_VERBOSE)
     set(EXTERNAL_VERBOSE_FLAG "-v")
 endif()
 
+if (JOBS)
+    list(APPEND NINJA_JOBS "-j ${JOBS}")
```

Comment:
I think it should be `list(APPEND NINJA_JOBS -j ${JOBS})` to not glue together `-j` and `jobs` into one argument.

Reply:
Gotcha thanks